src/core/include/openvino/core/preprocess/color_format.hpp

```diff
@@ -18,6 +18,7 @@ enum class ColorFormat {
     I420_THREE_PLANES,
     RGB,
     BGR,
+    GRAY,
```

Comment:
please, add documentation for enum value (s)

Reply:
Added documentation to be handled by doxygen